bool start_planning_session
float32 trench_x
float32 trench_y
float32 trench_d
bool delete_prev_traj
---
bool success
string message
